add readme (#12)
thetribeio/html-to-pdf: README.md

> # html-t-pdf

Reviewer: ```suggestion
# html-to-pdf
```

Author: rooh la boulette

> this is a api that can be used to generate a pdf from a source file.

Reviewer: ```suggestion
this is an API that can be used to generate a PDF from a source file.
```

## how to use

### local

install packages
```
yarn
```

run server

```
yarn start
```

### with docker
run `docker-compose`
```
docker-compose up -d
```

### api

server will start on port `3000`.

> You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a pdf for you.

Reviewer: ```suggestion
You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a PDF for you.
```

> You then will be able to make a `POST` request to `localhost:3000` with plain text data and the response will generate a pdf for you.

Reviewer: ```suggestion
You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a pdf for you.
```
